.github/workflows/test.yml

```
@@ -150,15 +153,15 @@ jobs:
           HLS_WRAPPER_TEST_EXE: hls-wrapper
         run: cabal test wrapper-test --test-options="$TEST_OPTS --rerun-log-file .tasty-rerun-log-wrapper"
 
-      - if: matrix.test && matrix.ghc != '9.2.4' && matrix.ghc != '9.4.2'
+      - if: matrix.test && matrix.ghc != '9.2.4' && matrix.ghc != '9.4.2' && matrix.ghc != '9.4.3'
```

We don't need to filter 9.4.2 out since there will be no test run for it. Please apply to all other lines in the PR.
```suggestion
      - if: matrix.test && matrix.ghc != '9.2.4' && matrix.ghc != '9.4.3'
```

---

Thank you - done.

Do you think the same should be some with GHC-9.2.5? Drop 9.2.4 and replace it with 9.2.5?